tests/trainer/test_trainer.py

```diff
@@ -193,12 +198,37 @@ def test_loading_meta_tags(tmpdir):
     logger.log_hyperparams(hparams)
     logger.save()
 
-    # load tags
+    # load hparams
     path_expt_dir = tutils.get_data_path(logger, path_dir=tmpdir)
+    hparams_path = os.path.join(path_expt_dir, 'hparams.yaml')
```

Comment:
use here the default
```suggestion
    hparams_path = os.path.join(path_expt_dir, TensorBoardLogger.NAME_HPARAMS_FILE)
```